jest.config.js

```diff
@@ -11,6 +11,8 @@ module.exports = {
   fakeTimers: { enableGlobally: true },
   transformIgnorePatterns: ['<rootDir>/node_modules/'],
   testPathIgnorePatterns: ['<rootDir>/lib/', '/node_modules/', '/examples/'],
+  silent: true,
+  verbose: false,
```

Comment:
I turned these off because there's a lot of console output when running tests which adds a lot of noise to the test output and makes it hard to find what really went wrong.